plugins/modules/vault_pki_generate_certificate.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2022, Florent David <florent.david@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+  module: vault_pki_generate_certificate
+  version_added: 2.11.6
```

Comment:
```suggestion
  version_added: 2.3.0
```
this needs to be collection version, and next expected for us is `2.3.0`

Reply:
Fixed in commit 3a81a4ba8da0bef037f518023c3e4553e7c66cda